cinema/views.py

```diff
@@ -17,28 +18,39 @@
     MovieSessionDetailSerializer,
     MovieListSerializer,
     OrderSerializer,
-    OrderListSerializer,
+    # OrderListSerializer,
```

Comment:
The `OrderListSerializer` is commented out in the import section. If this serializer is intended to be used, ensure that it is properly defined and uncommented where necessary.